UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -836,6 +836,16 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
         /// </summary>
         /// <param name="format">The format string.</param>
         /// <returns>The string representation.</returns>
+        /// <remarks>
```

Comment:
To avoid repeating xmldoc for method overloads in one type we can use `/// <inheritdoc cref="SomeMethod" />`.  Then you only need to specify the extra parameters or override any elements that need a different text.

We can also do the same to avoid repeating for N quantities, but maybe that's overdoing it. It's also more readable on github to add it to all quantities.

Reply:
Done. I wonder if we should be using         /// <see cref="QuantityFormatter.Format{TUnitType}(IQuantity{TUnitType}, string, IFormatProvider)"/> rather than inheritdoc though for cross-referencing though?